flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -708,6 +708,20 @@ public class TaskManagerOptions {
                             "Time we wait for the timers in milliseconds to finish all pending timer threads"
                                     + " when the stream task is cancelled.");
 
+    public static final ConfigOption<TaskManagerLoadBalanceMode> TASK_MANAGER_LOAD_BALANCE_MODE =
+            ConfigOptions.key("taskmanager.load-balance.mode")
+                    .enumType(TaskManagerLoadBalanceMode.class)
+                    .defaultValue(TaskManagerLoadBalanceMode.NONE)
+                    .withDescription(
+                            "The load balance mode of taskmanager when processing scheduling tasks.");
```

Comment:
It's better to describe these values in detail, it's useful for users to understand this option.

Reply:
👌